composer/utils/object_store/uc_object_store.py

```diff
@@ -206,13 +207,14 @@ def get_object_size(self, object_name: str) -> int:
         """
         from databricks.sdk.core import DatabricksError
         try:
-            file_info = self.client.files.get_status(self._get_object_path(object_name))
-            if file_info.is_dir:
-                raise IsADirectoryError(f'{object_name} is a UC directory, not a file.')
-
-            assert file_info.file_size is not None
-            return file_info.file_size
+            # Note: The UC team is working on changes to fix the files.get_status API, but it currently
+            # does not work. Once fixed, we will call the files API endpoint. We currently only use this
+            # function in Composer and LLM-foundry to check the UC object's existance.
+            self.client.api_client.do(method='HEAD',
```

Comment:
actually sorry, we should return a nubmer here, yeah?

Reply:
🔥